ExampleApp/Shared/PaymentSettingTableViewController.swift

```diff
@@ -95,6 +95,7 @@ class PaymentSettingTableViewController: UITableViewController {
     @IBOutlet var installmentKTCPaymentCell: UITableViewCell!
     @IBOutlet var installmentKBankPaymentCell: UITableViewCell!
     @IBOutlet var installmentSCBPaymentCell: UITableViewCell!
+    @IBOutlet var mobileBankingSCBPaymentCell: UITableView!
```

Comment:
Should be `UITableViewCell` and not `UITableView`.

Reply:
Updated, thank you!